posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -887,6 +939,73 @@ public class PostHog private constructor(
         replayQueue?.flush()
     }
 
+    /**
+     * Sets person properties that will be included in feature flag evaluation requests.
+     *
+     * @param properties Dictionary of person properties to include in flag evaluation
+     * @param reloadFeatureFlags Whether to automatically reload feature flags after setting properties
+     */
+    public fun setPersonPropertiesForFlags(
+        properties: Map<String, Any>,
+        reloadFeatureFlags: Boolean = true,
+    ) {
+        if (!isEnabled()) return
+        if (!hasPersonProcessing()) return
+        if (properties.isEmpty()) return
+
+        remoteConfig?.setPersonPropertiesForFlags(properties)
+
+        if (reloadFeatureFlags && this.reloadFeatureFlags) {
+            this.reloadFeatureFlags()
+        }
+    }
+
+    /**
+     * Resets all person properties that were set for feature flag evaluation.
+     */
+    public fun resetPersonPropertiesForFlags() {
+        if (!isEnabled()) return
+        if (!hasPersonProcessing()) return
+
+        remoteConfig?.resetPersonPropertiesForFlags()
+    }
```

Comment:
we should `reloadFeatureFlags` here as well right?

Reply:
I think you're right. This is a deviation from the iOS implementation. Similar to the other methods added here, I've added a `reloadFeatureFlags` boolean (default to true) which determines whether or not flags will be reloaded upon calling the method.